### PR TITLE
Caesar Cipher Added

### PR DESCRIPTION
**Pull request by Ra00f1**
That was a fun problem to solve, Thank you for posting it. :)

### CODE REVIEW

File: scripts/Caesar Cipher.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Cipher = {
```
Comment (GiovanniBruno22) [inline scripts/Caesar Cipher.py]: Can we rename the file so it does not contain any space? 

caesar_cipher.py

File: scripts/Caesar Cipher.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "V": "S",
+  "W": "T",
+  "X": "U",
+  "Y": "V",
+  "Z": "W"
+}
+
+if __name__ == "__main__":
+
+  print("Please enter a text to cipher: ")
```
Comment (GiovanniBruno22) [inline scripts/Caesar Cipher.py]: Minor: can we move lines 32 to 45 into a "main" function?

File: scripts/Caesar Cipher.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "T": "Q",
+  "U": "R",
+  "V": "S",
+  "W": "T",
+  "X": "U",
+  "Y": "V",
+  "Z": "W"
+}
+
+if __name__ == "__main__":
```
Comment (GiovanniBruno22) [inline scripts/Caesar Cipher.py]: Minor: can we move this at the end of the file?

**Review by GiovanniBruno22 — CHANGES_REQUESTED** [review]

**Review by GiovanniBruno22 — APPROVED** [review]
Nice! thank you so much for your contribution! 🙂 👍🏻

**Comment by Ra00f1** [comment]
Thank you for reviewing, I see those minor changes you suggested a lot but I tend to forget them a lot as well.

**Pull request merged**